.github/workflows/release-homebrew.yaml

```diff
@@ -0,0 +1,164 @@
+# When a third-party action is added (i.e., `uses`), please also add it to `download-licenses` in Makefile.
+# When the installers are ready in installer private bucket, run this workflow to test them.
+# TODO: Add job of uploading installers to Github release when installer tests are stable.
+# "release" runner tag is to make target hosts deterministic for easy clean up.
+# TODO: Remove the "release" runner tag when installer tests are stable.
+name: Release Homebrew
+on:
+  workflow_dispatch:
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  get-latest-tag:
+    name: Get the latest release tag
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.latesttag.outputs.tag }}
+      version: ${{ steps.latestversion.outputs.version }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
+      - name: 'Get the latest tag'
+        id: latesttag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: v0.1.0
+      - name: 'Convert tag to version'
+        id: latestversion
+        run: |
+          version=${{ steps.latesttag.outputs.tag }}
+          if [[  $version == v* ]]; then
+            version="${version:1}"
+          fi
+          echo "version=$version" >> ${GITHUB_OUTPUT}
+
+  # TODO: fix the arm64 test installer.
+  # Currently the first time of calling any Finch command in arm64 hosts triggered by Github action will fail by
+  # the error "Error: Process completed with exit code 137." So the arm64 job will fail.
+  # We temporarily use follow-up manual steps to complete and clean up it.
+  macos-arm64-test-installer:
+    needs: get-latest-tag
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          [
+            [self-hosted, macos, arm64, 11.7, release],
+            [self-hosted, macos, arm64, 12.6, release],
+            [self-hosted, macos, arm64, 13.0, release],
+          ]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    env:
+      ACCESS_TOKEN: ${{ secrets.FINCH_BOT_TOKEN }}
```

Comment:
@ningziwen Could you confirm what the env var ACCESS_TOKEN is used for? The release-installer.yaml also contains this. https://github.com/runfinch/finch/blob/main/.github/workflows/release-installer.yaml#L34

Reply:
This is needed for in your last step to raise the Homebrew PR but looks like you directly use it in line. So env is not needed. For installer test, it will be used to cut Github release. https://github.com/runfinch/finch/blob/main/.github/workflows/release-installer.yaml#L3